public/js/controllers/address_controller.js

```diff
@@ -52,6 +52,10 @@ function amountFlowProcessor (d, binSize) {
   }
 }
 
+function numberWithCommas (x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+}
```

Comment:
I figured there would be something in `public/js/helpers/humanize_helper.js`, but there isn't.  But as much as I respect the regexp you've got here, I think a more robust way would be with [`toLocaleString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat/NumberFormat).